Aws2-s3 tests fail on FIPS machine.

When running tests for `aws2-s3` on FIPS machine. Following failures can e observed:

```
Caused by: sun.security.pkcs11.wrapper.PKCS11Exception: CKR_ARGUMENTS_BAD
at jdk.crypto.cryptoki/sun.security.pkcs11.wrapper.PKCS11.C_GenerateRandom(Native Method)
at jdk.crypto.cryptoki/sun.security.pkcs11.P11SecureRandom.implNextBytes(P11SecureRandom.java:181)
... 14 more
```
or
```
Caused by: sun.security.pkcs11.wrapper.PKCS11Exception: CKR_DEVICE_ERROR
at jdk.crypto.cryptoki/sun.security.pkcs11.wrapper.PKCS11.C_GenerateRandom(Native Method)
at jdk.crypto.cryptoki/sun.security.pkcs11.P11SecureRandom.implNextBytes(P11SecureRandom.java:181)
... 99 more
```

--- a/integration-test-groups/aws2/aws2-s3/src/test/java/org/apache/camel/quarkus/component/aws2/s3/it/Aws2S3Test.java
+++ b/integration-test-groups/aws2/aws2-s3/src/test/java/org/apache/camel/quarkus/component/aws2/s3/it/Aws2S3Test.java
@@ -20,6 +20,7 @@ import java.net.URI;
 import java.net.URLDecoder;
 import java.nio.charset.StandardCharsets;
 import java.util.Locale;
+import java.util.Random;
 import java.util.UUID;
 import java.util.stream.Stream;
 
@@ -195,8 +196,8 @@ class Aws2S3Test extends BaseAWs2TestSupport {
     @Test
     public void upload() throws Exception {
         final String oid = UUID.randomUUID().toString();
-        final String content = RandomStringUtils.randomAlphabetic(8 * 1024 * 1024);
-
+        //pseudo random data are fine -> if secureRandom is used in FIPS -> sun.security.pkcs11.wrapper.PKCS11Exception: CKR_DEVICE_ERROR
+        final String content = RandomStringUtils.random(8 * 1024 * 1024, 0, 0, true, true, null, new Random());
         try {
             RestAssured.given()
                     .contentType(ContentType.TEXT)

--- a/integration-tests-support/aws2/src/test/java/org/apache/camel/quarkus/test/support/aws2/Aws2TestResource.java
+++ b/integration-tests-support/aws2/src/test/java/org/apache/camel/quarkus/test/support/aws2/Aws2TestResource.java
@@ -89,6 +89,8 @@ public final class Aws2TestResource implements QuarkusTestResourceLifecycleManag
                     .withServices(services);
             localstack.withEnv("LS_LOG", localstackLogLevel);
             localstack.withEnv("PROVIDER_OVERRIDE_CLOUDWATCH", "v1");
+            localstack.withEnv("AWS_ACCESS_KEY_ID", "testAccessKeyId"); //has to be longer then `test`, to work on FIPS systems
+            localstack.withEnv("AWS_SECRET_ACCESS_KEY", "testSecretKeyId");
             localstack.withLogConsumer(new Slf4jLogConsumer(LOG));
             localstack.start();
 

